vkconfig/layer_info.json

```diff
@@ -366,6 +366,35 @@
                 "type": "save_file",
                 "default": "stdout"
             }
+        },
+        "VK_LAYER_LUNARG_screenshot": {
+            "frames": {
+                "name": "Frames",
+                "description": "Comma separated list of frames to output as screen shots or a range of frames with a start, count, and optional interval separated by a dash. Setting the variable to \"all\" will output every frame. Example: \"5-8-2\" will output frame 5, continue until frame 13, dumping every other frame. Example: \"3,8-2\" will output frames 3, 8, and 9.",
+                "type": "string",
+                "default": "0-0"
```

Comment:
Does this mean it will take a screenshot of frame 0?  Could we make the default be the empty string?

Reply:
It will take a screenshot of frame 0 which is just the first frame.
I can make the default and empty string.